### DISCUSSION TITLE
Can this extension be provided to chromium browsers?

### DISCUSSION THREAD

**Discussion by sf1tzp** [AsciiJakob/NoDistractions#19] [General]
Hi! This extension suits my needs perfectly at home, but at work I have to use edge browser. Have you considered making this available to those browsers as well?

Sorry I don't know how much effort is required to do so. Thanks for all your work on this extension so far :)

**Comment by AsciiJakob** [comment]
Hey! Thank you so much for the feedback, I'm glad to hear you're enjoying the extension. I have opened an issue about this (#20) since I think it may be worth to look into.

It's supposed to be trivial to have cross-browser extensions, [but there are some incompatibilities which might make things a bit annoying ](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities)(such as no support for promises on chrome with manifest v2). Saying that actually, I just remembered that manifest v2 extensions are getting deprecated on chrome, so this suddenly got a lot harder; I might have to move to manifest v3 which I'm not sure is possible with how the extension works and which might take a lot of time. Anyway thanks for expressing interest, I'm gonna look into this sometime, hopefully everything works out
